src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,54 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName("Monica");
+        manufacturer.setCountry("Mexico");
+        Manufacturer manufacturerMonica = manufacturerService.create(manufacturer);
+        System.out.println(manufacturerMonica);
+
+        long manufacturerId = 11L;
+        System.out.println(manufacturerService.get(manufacturerId));
+
+        List<Manufacturer> allManufacturers = manufacturerService.getAll();
+        allManufacturers.forEach(System.out::println);
+
+        Manufacturer updateManufacturer = new Manufacturer(7L, "Chandler", "Britain");
```

Comment:
What is 7L? If I will clone your project and run the main method everything will work ok?